index.js

```diff
@@ -125,12 +124,34 @@ var torrentStream = function (link, opts, cb) {
     }
   })
 
+  var handleReserved = function (p) {
+    var s = p.split(path.sep)
+    var root = path.parse(p).root
+    var n = []
+
+    if (s.length === 0) {
+      return ''
+    }
+
+    var i = 0
+    if (s[0] + path.sep === root) {
+      i = 1
+      n.push(s[0])
+    }
+    while (i < s.length) {
+      n.push(s[i].replace(/[*:/\\"><?|]/g, '_'))
+      ++i
+    }
+    return n.join(path.sep)
+  }
```

Comment:
Does it have to be this complicated? Can't you just do `split()` -> `map()` -> `replace()` -> `join()`?